js/index.js

```diff
@@ -0,0 +1,38 @@
+function getRandomInt(min, max) {
+	min = Math.ceil(min);
+	max = Math.floor(max);
+	return Math.floor(Math.random() * (max - min + 1)) + min;
+}
+let dice = {
+	value: 1,
```

Comment:
You can actually assign the initial value of this dice to a random number as well using the `getRandomInt()` function. That way the dice will always start with a new random number, which will enable for an easier refresh.

Reply:
i did't what you ask for  but in this case i have to click on the button to see the dice